src/js/services/opencapService.js

```diff
@@ -0,0 +1,160 @@
+'use strict';

```

Comment:
1. Your service doesn't follow our standards.
https://github.com/Bitcoin-com/Wallet/blob/master/src/js/services/shapeshift.service.js
The link below is an example of our standard for a service.
2. If you want to use a promise, use the promise from Angular1 provided by `$q`.
Example
`
var deferred = $q.defer();
...
deferred.resolve(res);
deferred.reject(err);
...
return deferred.promise;
`
3. If you want make a http request, please use `$http` from Angular1 instead of `fetch`.
4. file naming convention `opencap.service.js`
5. declare your module in `bitcoincom.services`

Reply:
Perfect. I'll work on this, thanks!

Reply:
Just finished the update. I covered all of your points, let me know if there is anything else you want changed!

Reply:
Thanks, I will review it and let you know. I don't guarantee anything, I will review technically if it is suitable to our app.

Reply:
Sounds good. I'll keep an eye if you need more changes made.